### ISSUE TITLE
Tab displays Gmail icon while loading

### ISSUE THREAD

**Issue by hairlessbear** [HabitRPG/habitica#5002]
When loading the HabitRPG page fresh (i.e., not a reload), the tab always displays the Gmail icon at least once before switching to the proper icon. [Here's a video of the behavior](https://hairlessbear.tinytake.com/sf/MTE1OTY3Xzc1ODE3NA).

I've only tested this in Chrome, and clearing the cache doesn't fix the issue.

**Comment by Alys** [comment]
Oh that looks really weird!  

I can't replicate it though, even if I load HabitRPG into a tab that previously contained Gmail. I'm wondering if it's a weird thing that your particular browser install is doing.

Do you have a bookmark for HabitRPG? Could you please look for one, and if there is one, see what icon it has? If it's the Gmail icon, could you delete the bookmark and then load HabitRPG again? If deleting the bookmark and reloading doesn't help, try clearing your cache again, then closing all browser windows, and then loading HabitRPG.

**Comment by gisikw** [comment]
Extracted from the page source, it looks like we've got a broken favicon link:

`<link rel="shortcut icon" href="/images/favicon-8b58f6a1.ico">`

https://habitrpg.com/images/favicon-8b58f6a1.ico doesn't resolve.

Note that http://habitrph.com/favicon.ico _does_ resolve, and most browsers will use this as a sensible default. But it it possible Chrome in this case is just using the gmail icon as a fallback for the broken /images/favicon-8b58f6a1.ico request?

**Comment by hairlessbear** [comment]
@Alys I don't have HabitRPG as a bookmark, but I did have one for Gmail (well, three actually; apparently I'm a bookmark hoarder). I deleted the Gmail bookmarks and tried what you said, but no dice. I also tried running CCleaner to no avail. I also tested this in Firefox, and couldn't replicate the issue. 

For what it's worth, I've been having some other little performance issues in Chrome recently, so I wouldn't be surprised if this issue is exclusive to me and my install.
